### DISCUSSION TITLE
text not displaying clearly especially in day time photos

### DISCUSSION THREAD

**Discussion by baryah** [aaronwmorris/indi-allsky#602] [Q&A] [answered]
I have tried different colours and sizes, but at day time the text appears as in the attached file. It appears fine if I choose black color (i.e.0, 0, 0). Can we have different colours during day and night or any other way to sort this out.

![Screenshot 2023-03-05 at 11 29 13 AM](https://user-images.githubusercontent.com/44144990/222944304-98338f77-68eb-4ea6-938d-ca72d59504bc.jpg)

**Comment by aaronwmorris** [comment, marked as answer]
Try the following settings for the text.  Basically doubling all of the values.

Most of what you are seeing there is the HTML5 canvas scaling the image, which does not always look very good with text.

![Screenshot_20230305_111040](https://user-images.githubusercontent.com/17464290/222972082-2616227d-9a70-4e26-b81b-40c443128874.png)

**Comment by dbainbridge** [comment]
What about overlaying a separate view on top of the same area that the image has the text displayed so that is can be normal text?

**Comment by baryah** [comment]
Thanks. That made it much better.
